kiosk/src/Models/Kiosk.ts

```diff
@@ -333,7 +333,8 @@ export class Kiosk {
             iframe.src = src;
             return iframe;
         }
-        gamespace.appendChild(createIFrame(playUrlBase + playQueryParam));
+        gamespace.appendChild(createIFrame(playUrlBase + playQueryParam)).focus();
```

Comment:
nit; usually I'd prefer to see this split up rather than actually use the return value; e.g.

```suggestion
        const playerIFrame = createIFrame(playUrlBase + playQueryParam);
        gamespace.appendChild(playerIFrame);
        playerIFrame.focus();
```

a bit easier for debugging (actually keeping a ref around that you can always log out or compare against), less meaningful things at the end of the line that might get missed when reading, and you don't need to have deep lore on apis (does `A.appendChild(B)` return `A` or `B`, since both are Nodes in the first place? Does it matter in this case / was the correct one selected in the first place?)